dazzle.yaml

```diff
@@ -155,13 +150,33 @@ combiner:
         - lang-java:17
     - name: yugabytedb
       ref:
-      - full
+      - base
```

Comment:
This is good ✨ . Wondering why we were using `full` before 🤔

Reply:
Most of the database images do, it's like an unofficial convention.